servicetalk-traffic-resilience-http/src/main/java/io/servicetalk/traffic/resilience/http/PeerRejectionPolicy.java

```diff
@@ -16,24 +16,65 @@
 package io.servicetalk.traffic.resilience.http;
 
 import io.servicetalk.capacity.limiter.api.CapacityLimiter;
-import io.servicetalk.capacity.limiter.api.RequestRejectedException;
+import io.servicetalk.capacity.limiter.api.RequestDroppedException;
+import io.servicetalk.circuit.breaker.api.CircuitBreaker;
 import io.servicetalk.http.api.HttpResponseMetaData;
+import io.servicetalk.http.api.HttpResponseStatus;
 import io.servicetalk.http.api.StreamingHttpResponse;
 
 import java.time.Duration;
 import java.util.function.Function;
 import java.util.function.Predicate;
 
-import static io.servicetalk.traffic.resilience.http.PeerCapacityRejectionPolicy.Type.REJECT;
-import static io.servicetalk.traffic.resilience.http.PeerCapacityRejectionPolicy.Type.REJECT_PASSTHROUGH;
-import static io.servicetalk.traffic.resilience.http.PeerCapacityRejectionPolicy.Type.REJECT_RETRY;
+import static io.servicetalk.http.api.HttpResponseStatus.BAD_GATEWAY;
+import static io.servicetalk.http.api.HttpResponseStatus.SERVICE_UNAVAILABLE;
+import static io.servicetalk.http.api.HttpResponseStatus.TOO_MANY_REQUESTS;
+import static io.servicetalk.traffic.resilience.http.PeerRejectionPolicy.Type.REJECT;
+import static io.servicetalk.traffic.resilience.http.PeerRejectionPolicy.Type.REJECT_PASSTHROUGH;
+import static io.servicetalk.traffic.resilience.http.PeerRejectionPolicy.Type.REJECT_RETRY;
 import static java.time.Duration.ZERO;
 import static java.util.Objects.requireNonNull;
 
 /**
  * Policy for peer capacity rejections that allows customization of behavior (retries or pass-through).
+ * This is meant to be used as a policy on the {@link TrafficResilienceHttpServiceFilter}.
+ * @see TrafficResilienceHttpClientFilter.Builder#peerRejection(PeerRejectionPolicy)
  */
-public final class PeerCapacityRejectionPolicy {
+public final class PeerRejectionPolicy {
```

Comment:
After looking at new names and javadoc, will it make sense to name them as `ClientRejectionPolicy` and `ServiceRejectionPolicy` to align with corresponding filters they are created for? Or maybe `ClientPeerRejectionPolicy` and `ServicePeerRejectionPolicy`?